boards/arm/arduino_opta_m4/doc/index.rst

```diff
@@ -0,0 +1,170 @@
+.. _arduino_opta_m4_board:
+
+Arduino OPTA M4-Core
+#######################
+
+Overview
+********
+
+Opta™ is a secure micro Programmable Logic Controller (PLC)
+with Industrial Internet of Things (IoT) capabilities.
+
+Developed in partnership with Finder®, this device supports both the Arduino
+programming language and standard IEC-61131-3 PLC programming languages,
+such as Ladder Diagram (LD), Sequential Function Chart (SFC),
+Function Block Diagram (FBD), Structured Text (ST), and Instruction List (IL),
+making it an ideal device for automation engineers.
+
+For Zephyr RTOS, only the M4 is supported for now, making the M7 to run the PLC
+tasks while the M4 core under Zephyr acts as a coprocessor.
+
+Additionally, the device features:
```

Comment:
The feature list looks like a copy paste mistake from the EVK :slightly_smiling_face: